xscen/aggregate.py

```diff
@@ -796,8 +1068,9 @@ def produce_horizon(
             ds_merge = xr.Dataset()
             for freq, ds_ind in ind_dict.items():
                 if freq != "fx":
-                    ds_mean = climatological_mean(
+                    ds_mean = climatological_op(
```

Comment:
I would instead add `op` to the list of parameters for `produce_horizon`, with a default of `mean`.

Reply:
Also, you currently fail the tests for `produce_horizon` because of `rename_variables`.

Reply:
> I would instead add `op` to the list of parameters for `produce_horizon`, with a default of `mean`.

Let's do that separately from this PR.

Reply:
Sure. In that case, could you just put a `ToDo` so we don't forget?

Reply:
ToDo added.